lib/components/Atom/WheelPicker/base/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { HTMLAttributes, useEffect, useMemo, useRef } from "react";
+import clsx from "clsx";
+import { DropdownItem } from "@components/Atom/dropdown";
+import "./wheel-picker.scss";
+
+export interface WheelPickerProps extends HTMLAttributes<HTMLElement> {
+    data: { value: string | number }[];
+    selectedValue: string | number;
+    setSelectedValue: (value: string | number) => void;
+    handleKeyDown?: (e: React.KeyboardEvent) => void;
+    isFocused?: boolean;
+    dropDownItemClassName?: string;
+    containerClassName?: string;
+    listClassName?: string;
+    position?: "first" | "center" | "last" | undefined;
```

Comment:
Can we make use of `TPosition` here?

Reply:
I will use THorizontalPosition, that is a better suit